src/Dialog/Content/Panel.tsx

```diff
@@ -97,8 +98,8 @@ const Panel = React.forwardRef<ContentRef, PanelProps>((props, ref) => {
   let closer: React.ReactNode;
   if (closable) {
     closer = (
-      <button type="button" onClick={onClose} aria-label="Close" className={`${prefixCls}-close`}>
-        {closeIcon || <span className={`${prefixCls}-close-x`} />}
+      <button type="button" onClick={onClose} title={closeLabel} aria-label={closeLabel ?? "Close"} className={`${prefixCls}-close`}>
```

Comment:
should be a prop with `closable`

Reply:
do you mean make the closable prop an object with prop label?

Reply:
Yeah, it might look like this:
`closable: boolean | { icon?: React.ReactNode, ariaLabel?: string }` + old `closeIcon`.

Reply:
https://github.com/ant-design/ant-design/discussions/42828